app/components/details/template.hbs

```diff
@@ -4,7 +4,8 @@
   <div class="details-wrapper">
     <div class="card">
       <h1 class="flexible">{{this.model.title}}</h1>
-      <img src="{{this.model.imageURL}}" class="rounded">
+      {{!-- template-lint-disable require-valid-alt-text --}}
+      <img alt='project image' src="{{this.model.imageURL}}" class="rounded">
```

Comment:
Change content `alt` to just `project` should be resolve problem with linter and you will able to remove this comment :)